services/paymaster/.sqlx/query-8adbc3bfb60a7802966a144aa94271828586b1f4a9f393e034993cb1405904e4.json

```diff
@@ -30,21 +30,13 @@
           "Custom": {
             "name": "network_environment",
             "kind": {
-              "Enum": [
-                "mainnet",
-                "testnet"
-              ]
+              "Enum": ["mainnet", "testnet"]
```

Comment:
We should probably ignore this entire directory from biome given this is managed by rust tooling & these files in particular are generated by sqlx